Makefile

```diff
@@ -50,6 +50,11 @@ else
 printfile = cat $(1)
 endif
 
+ifdef WINDOWS
+RM = cmd /C del /Q /F $(subst /,\,$(1))
+RMDIR  = cmd /C rmdir /S /Q $(subst /,\,$(1))
+endif
```

Comment:
You can make non-Windows versions of these, and then you won't have to use Windows-specific logic in the `clean` target. So, add an `else` clause and just define:
```makefile
else
RM = rm -f $(1)
RMDIR = rm -rf $(1)
endif
```